### PR TITLE
Describe how to set the timezone for the build VM.

### PR DESCRIPTION
**Pull request by bbewan**
https://app.clubhouse.io/buddybuild/story/6900/describe-how-to-set-the-vm-s-timezone

### CODE REVIEW

**Comment by bbdocs** [comment]
See the website here: http://123-bb-docs.surge.sh

File: builds/faq.adoc [hunk 1]
```diff
@@ -32,3 +32,28 @@ make builds for old branches less visible:
...
+== How do I configure the timezone in the build VM?
+
+In the link:custom_build_steps.adoc#postclone[`buddybuild_postclone.sh`]
+script, add the following line (depending on the type of build):
+
+- **iOS builds**
++
+[source,bash]
+----
+sudo systemsetup -settimezone America/Toronto
```
Comment (nathanshox) [inline builds/faq.adoc]: Does this work? I had to do `echo password | sudo -S systemsetup -settimezone America/Toronto` to get it to work when I was doing it before?

Comment (bbewan) [reply builds/faq.adoc]: When I use "Rebuild with Remote Access", I can `sudo bash` without a password, so I presume that `sudo` without a password works in all cases. If it didn't, then we'd have to tell customers what password we're using. That seems a bit silly; why have a password if it's public?

Comment (nathanshox) [reply builds/faq.adoc]: I don't think we allow you to SSH into the box via a password (@andrew, can you confirm?), so we're not opening up a security risk by allowing it. The password is `password` afterall, so its not even a good one :)

One way to test if it works would be to check the timezone has actually been changed in a prebuild script.

Comment (nathanshox) [reply builds/faq.adoc]: I'm testing it out here on one of my projects https://alphaadmin.buddybuild.com/#!/builds/5a174025a9b9ce00010446b8

Comment (nathanshox) [reply builds/faq.adoc]: It does work 🎉 

```
=== Running Custom Post-Clone Step ===
2017-11-23T21:50:05.356Z - ci:     ### BEGIN buddybuild_postclone.sh ###
2017-11-23T21:50:05.358Z - ci:     Changing timezone to Toronto
2017-11-23T21:50:05.361Z - ci:     Current Time:  Thu Nov 23 13:50:05 PST 2017
2017-11-23T21:50:05.426Z - ci:     Set TimeZone: America/Toronto
2017-11-23T21:50:15.446Z - ci:     Current Time:  Thu Nov 23 16:50:15 EST 2017
2017-11-23T21:50:15.448Z - ci:     ### END buddybuild_postclone.sh ###
...
=== Running Custom Pre-Build Step ===
2017-11-23T21:54:14.067Z - ci:     ### BEGIN buddybuild_prebuild.sh ###
2017-11-23T21:54:14.072Z - ci:     Current Time:  Thu Nov 23 16:54:13 EST 2017
2017-11-23T21:54:14.087Z - ci:     ### END buddybuild_prebuild.sh ###
```

**Pull request merged**